se/se_epub_lint.py

```diff
@@ -2573,14 +2573,14 @@ def lint(self, skip_lint_ignore: bool) -> list:
 	missing_selectors = []
 	single_use_css_classes = []
 
-	for css_class in xhtml_css_classes:
-		if css_class not in IGNORED_CLASSES:
-			if f".{css_class}" not in self.local_css:
-				missing_selectors.append(css_class)
+	for css_class in xhtml_css_classes.items():
```

Comment:
Can we do something like `for css_class, count in xhtml_css_classes.items()` so it's clearer instead of using subscripts in the looop?

Reply:
Sounds reasonable, I was wondering how best to do that in Python.